README.md

```diff
@@ -76,7 +76,7 @@ Optional Environment Variables:
     - example: `{"MO": ["SAINT LOUIS"], "KS": []}`
   - `CVS_BLOCK_LIST`: (optional): JSON of states and cities to not be warned about new city for.
     - example: `{"MO": ["SAINT LOUIS"], "KS": []}`
-  Any city that CVS returns for the state not in either of those two lists will log a warning to add it to one.
+  - Any city that CVS returns for the state(s) in the allow list that are not listed in the block list will cause a warning message to be logged. Then it may be added to the allowed / blocked list depending on if you wish to have the locations in that city checked or not checked.
```

Comment:
minor, it's if not in either list
```suggestion
  - Any city that CVS returns for the state(s) in the allow list that are not listed in the allow list or block list will cause a warning message to be logged. Then it may be added to the allowed / blocked list depending on if you wish to have the locations in that city checked or not checked.
```